Algorithm Codes/Graph/AdjacencyMatrixGraph.py

```diff
@@ -42,13 +46,42 @@ def dijkstra(self, start):
                         heapq.heappush(heap, (dist[i], i))
             
         return (dist, prev)
+    
+    def bellman_ford(self, start):
+        dist = [float('inf') for i in range(self.n)]

```

Comment:
When creating a list of same element in array in Python,
How about this code instead? :) 
`[float('inf')] * self.n`

Reply:
I see this in other part of the codes :D

Reply:
TBH, I like my way of doing it lol.

Reply:
Haha then let's keep it that way :)